securedrop_client/gui/auth/sign_in/error_bar.py

```diff
@@ -31,6 +32,8 @@ def __init__(self) -> None:
         super().__init__()
 
         self.setObjectName("LoginErrorBar")
+        styles = resource_string(__name__, "error_bar.css").decode("utf-8")
```

Comment:
The `error_bar.css` file was introduced in https://github.com/freedomofpress/securedrop-client/pull/1380/files#diff-ee064a58e8d28e31d3616af7b5dfcbccc06587b3bc0a23b6fddabecdadb7deb1, which is why the diff in this PR does not show it being added.

Reply:
Correct. This commit message says "Fix..." because I didn't remove the corresponding CSS from `sdclient.css` I completely missed that I wasn't using the new stylesheet :grimacing: